benchconnect/benchconnect/_post.py

```diff
@@ -0,0 +1,84 @@
+from json import load, loads
+from pathlib import Path
+
+import click
+from benchclients.conbench import ConbenchClient
+
+
+def post_blob(json: dict, endpoint: str, client: ConbenchClient) -> None:
+    "Post a blob of JSON to Conbench"
+    client.post(path=endpoint, json=json)
+
+
+def post_file(path: str, client: ConbenchClient, endpoint: str) -> None:
+    "Load a JSON file and post it to Conbench"
+    with open(Path(path).resolve(), "r") as f:
+        json = load(f)
+
+    post_blob(json=json, endpoint=endpoint, client=client)
+
+
+def poster(json: str, path: str, endpoint: str) -> None:
+    "Take either a blob or a path and post the resulting JSON to Conbench"
+    client = ConbenchClient()
+
+    if json:
+        post_blob(json=loads(json), endpoint=endpoint, client=client)
+
+    elif path and Path(path).resolve().is_file():
+        post_file(path=path, client=client)
+
+    elif path and Path(path).resolve().is_dir():
+        for file in Path(path).resolve().glob("*.json"):
+            post_file(path=file, endpoint=endpoint, client=client)
+
+    else:
+        click.echo("No data to post found!")
+
+
+@click.command(
+    help="""
+Post benchmark result JSON[s] to a Conbench API
+
+Specify either `--json` or `--path`.
+
+For Conbench environment variables, see `benchconnect post --help`.
```

Comment:
Wonder if you could import the env var help string instead of pointing to a different help doc here?